test/e2e/utils/test_context.go

```diff
@@ -160,6 +160,8 @@ func (kc *KBTestContext) Make(makeOptions ...string) error {
 
 // CleanupImage is for cleaning up the docker images for testing
 func (kc *KBTestContext) Destroy() {
+	// nolint:gosec
+	// False positive: G204: Subprocess launched with function call as argument or cmd arguments (gosec)
```

Comment:
I'd drop this line, since it's not accurate. It is certainly risky to launch a subprocess. 
What if the docker binary has been replaced by an attacker somehow?

Reply:
Yes it true. It is not a false positive at all. However, I am not sure how we could fix it.
Have you any idea?

Reply:
I mean we only keep `// nolint:gosec` which is necessary.